cypress/e2e/legacy_invitation_usecase_page.cy.ts

```diff
@@ -88,7 +88,7 @@ describe('UseCase Page', () => {
         const record = response.body.find((x) => x.threadId === threadId && x.state === 'request-received')
         cy.request('POST', `${TEST_AGENT_URL}/proofs/${record.id}/accept-request`).then(() => {
           // eslint-disable-next-line cypress/no-unnecessary-waiting
-          cy.wait(3000) // wait for the test agent request to be processed
+          cy.wait(10000) // wait for the test agent request to be processed
```

Comment:
10000 is quite long. Can we make all `cy.wait` like 5000? 

This is an anti-pattern so we should try to avoid the use `cy.wait` wherever we can.